types/trianglify/index.d.ts

```diff
@@ -1,42 +1,100 @@
+import { type Color, mix, type Scale } from "chroma-js";
+
+declare function trianglify(opts?: trianglify.Options): trianglify.PatternInterface;
+
 declare namespace trianglify {
+    interface ColorFunctionInput {
+        xPercent: number;
+        yPercent: number;
+        xScale: Scale<Color>;
+        yScale: Scale<Color>;
+        opts: number[][];
+        random?: () => number | undefined;
+    }
+
+    type ColorFunction = (args: ColorFunctionInput) => ReturnType<typeof mix>;
+    type Points = Array<[number, number]>;
+
     interface Options {
         /** Width of pattern */
-        width?: number | undefined;
+        width?: number;
         /** Height of pattern */
-        height?: number | undefined;
+        height?: number;
         /** Size of the cells used to generate a randomized grid */
-        cell_size?: number | undefined;
+        cellSize?: number;
         /** how much to randomize the grid */
         variance?: number | undefined;
         /** Seed for the RNG */
         seed?: number | string | null | undefined;
         /** X color stops */
-        x_colors?: false | string | string[] | undefined;
+        xColors?: string[];
         /** Y color stops */
-        y_colors?: false | string | string[] | undefined;
+        yColors?: string[];
         /** Color space used for gradient construction & interpolation */
-        color_space?: string | undefined;
+        colorSpace?: "rgb" | "hsl" | "hsv" | "hsi" | "lab" | "oklab" | "lch" | "oklch" | "hcl" | "lrgb" | undefined;
         /** Color function f(x, y) that returns a color specification that is consumable by chroma-js */
-        color_function?: ((x: number, y: number) => string) | null | undefined;
+        colorFunction?: ColorFunction;
         /** Width of stroke. Defaults to 1.51 to fix an issue with canvas antialiasing. */
-        stroke_width?: number | undefined;
-        /** An array of [x,y] coordinates to trianglulate. Defaults to undefined, and points are generated. */
-        points?: number[] | undefined;
+        strokeWidth?: number | undefined;
+        /** An array of [x,y] coordinates to triangulate. Defaults to `undefined`, and points are generated. */
+        points?: Points | undefined;
+        /** A boolean value to tell if trianglify should render the triangles or not. Defaults to `true` */
+        fill?: boolean | undefined;
```

Comment:
I think this is inconsistent, defaults like width/height should not have the defaults, so no undefined as supported type, that looks OK. But here (and other places)  `fill`, won't have default `null`, if someone passes `undefined:
https://github.com/qrohlf/trianglify/blob/master/src/trianglify.js#L48

Reply:
You're right, my bad 🙈. I'll test every property to make sure they are consistent and I'll update this PR =)

Thanks for your review ❤️

Reply:
Done @peterblazejewicz =)